RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3556 
Task addr: 2147524556 
Task slot: 0 
Task slot TaskAddr: 2147524556 
2NI_RX DONE!
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 0 
API_Try2Send success!
4NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
6NI_RX DONE!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 0  from app 1 
taskSlot run: 0 
7NI_RX DONE!
Bank INIT
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
dynamicEnergy_Router >> 4020486 
total instructions~~~> 1999037 
TotalEnergy: 18447011 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
dynamicEnergy_Router >> 3963156 
total instructions~~~> 999958 
TotalEnergy: 44338101 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1000001 
TotalEnergy: 44366960 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3964832 
total instructions~~~> 1000102 
TotalEnergy: 44375934 
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3965122 
total instructions~~~> 999900 
TotalEnergy: 44384058 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1000001 
TotalEnergy: 44366904 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1000102 
TotalEnergy: 44370351 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3964370 
total instructions~~~> 999621 
TotalEnergy: 44331579 
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1000203 
TotalEnergy: 44395394 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 999736 
TotalEnergy: 44356919 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1000001 
TotalEnergy: 44367012 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 1 
Task total size (txt+bss): 1472 
Task addr: 2147565064 
Task slot: 1 
Task slot TaskAddr: 2147565064 
2NI_RX DONE!
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 4 
API_Try2Send success!
4NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um TASK_START!
6NI_RX DONE!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 0  from app 0 
taskSlot run: 1 
7NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
PROD1 Start 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
dynamicEnergy_Router >> 3973278 
total instructions~~~> 1000102 
TotalEnergy: 44904082 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3958385 
total instructions~~~> 1001718 
TotalEnergy: 44280574 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
dynamicEnergy_Router >> 3965757 
total instructions~~~> 998183 
TotalEnergy: 44330880 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3966971 
total instructions~~~> 1000141 
TotalEnergy: 44415508 
SendingQueue_front: 1 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
dynamicEnergy_Router >> 3966799 
total instructions~~~> 1001011 
TotalEnergy: 44453597 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3966027 
total instructions~~~> 998789 
TotalEnergy: 44380366 
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
dynamicEnergy_Router >> 3970688 
total instructions~~~> 1000203 
TotalEnergy: 44508635 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3966971 
total instructions~~~> 999799 
TotalEnergy: 44401395 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
dynamicEnergy_Router >> 3966799 
total instructions~~~> 1001169 
TotalEnergy: 44459398 
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3965377 
total instructions~~~> 998890 
TotalEnergy: 44371966 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 3 
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3964531 
total instructions~~~> 1008990 
TotalEnergy: 44721330 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 2 
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3971914 
total instructions~~~> 991012 
TotalEnergy: 44163680 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
dynamicEnergy_Router >> 3969227 
total instructions~~~> 1001112 
TotalEnergy: 44511810 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 7 
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3963892 
total instructions~~~> 1002246 
TotalEnergy: 44473955 
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3964151 
total instructions~~~> 996163 
TotalEnergy: 44194739 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
dynamicEnergy_Router >> 3973779 
total instructions~~~> 1000102 
TotalEnergy: 44592864 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3965757 
total instructions~~~> 999799 
TotalEnergy: 44357990 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
dynamicEnergy_Router >> 3966799 
total instructions~~~> 1001415 
TotalEnergy: 44486150 
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 6 
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3965107 
total instructions~~~> 1002324 
TotalEnergy: 44503540 
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
dynamicEnergy_Router >> 3970959 
total instructions~~~> 996365 
TotalEnergy: 44363626 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3966971 
total instructions~~~> 999698 
TotalEnergy: 44397016 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 257 
from app: 0 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 259 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 258 
from app: 0 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
dynamicEnergy_Router >> 3966799 
total instructions~~~> 1001314 
TotalEnergy: 44465779 
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
taskSlot: 1 
Adding a msg to task 1  in the PIPE slot 256 
from app: 0 
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
PROD1 End 
Finishing task:0 app: 0 
From slot: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 0 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3965757 
total instructions~~~> 1002699 
TotalEnergy: 44540422 
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
Checking the PIPE of taskSlot: 1 
i: 0 
status: 4294967295 
---
i: 1 
status: 4294967295 
---
i: 2 
status: 4294967295 
---
i: 3 
status: 4294967295 
---
Task 0 deleted with sucsess! From application 0 
TaskList[0 ]status: 1 
Returning because of: 0 TaskList[i].status 1 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
dynamicEnergy_Router >> 3964190 
total instructions~~~> 996163 
TotalEnergy: 44221918 
SendingQueue_front: 1 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
dynamicEnergy_Router >> 3958299 
total instructions~~~> 999698 
TotalEnergy: 44218416 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
dynamicEnergy_Router >> 3961119 
total instructions~~~> 1000203 
TotalEnergy: 44299218 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 7 
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
dynamicEnergy_Router >> 3963156 
total instructions~~~> 999900 
TotalEnergy: 44343477 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3961550 
total instructions~~~> 1000001 
TotalEnergy: 44310716 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3973450 
total instructions~~~> 1000001 
TotalEnergy: 44562247 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
dynamicEnergy_Router >> 3958933 
total instructions~~~> 1000102 
TotalEnergy: 44256466 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 5 
API_Try2Send success!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
8NI_RX DONE!
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 2 
from app: 1 
dynamicEnergy_Router >> 3964370 
total instructions~~~> 1007677 
TotalEnergy: 44681401 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
dynamicEnergy_Router >> 3954225 
total instructions~~~> 992181 
TotalEnergy: 43817536 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 1 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 1 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
dynamicEnergy_Router >> 3959905 
total instructions~~~> 1000001 
TotalEnergy: 44246331 
SendingQueue_front: 6 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 3 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 3 
from app: 1 
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 1 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
dynamicEnergy_Router >> 3956693 
total instructions~~~> 1000001 
TotalEnergy: 44223320 
SendingQueue_front: 2 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 2 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 1  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 2  in the PIPE slot 0 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 3  in the PIPE slot 0 
from app: 1 
dynamicEnergy_Router >> 3957515 
total instructions~~~> 999799 
TotalEnergy: 44216919 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 4 
Mensagem nÃ£o encontrada, adicionando ao PendingReq!
8NI_RX DONE!
dynamicEnergy_Router >> 3953834 
total instructions~~~> 1000203 
TotalEnergy: 44134795 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 4  in the PIPE slot 2 
from app: 1 
>>>>>Achei aqui no pending!
SendingQueue_front: 2 
API_Try2Send success!
Bank sys_Sended all patternsFinishing task:0 app: 1 
From slot: 0 
Checking the PIPE of taskSlot: 0 
i: 0 
status: 1 
---
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
Checking the PIPE of taskSlot: 0 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
Checking the PIPE of taskSlot: 0 
i: 0 
status: 1 
---
RX interruption catched
Chegou um message request! App: 1 Task: 3 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
Checking the PIPE of taskSlot: 0 
i: 0 
status: 4294967295 
---
i: 1 
status: 4294967295 
---
i: 2 
status: 4294967295 
---
i: 3 
status: 4294967295 
---
Task 0 deleted with sucsess! From application 1 
TaskList[0 ]status: 0 
TaskList[1 ]status: 0 
TaskList[2 ]status: 0 
TaskList[3 ]status: 0 
TaskList[4 ]status: 0 
TaskList[5 ]status: 0 
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3959227 
total instructions~~~> 999900 
TotalEnergy: 43222914 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953548 
total instructions~~~> 103525 
TotalEnergy: 8017284 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3124 
Task addr: 2147524556 
Task slot: 0 
Task slot TaskAddr: 2147524556 
2NI_RX DONE!
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 1 
API_Try2Send success!
4NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
6NI_RX DONE!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 4  from app 1 
taskSlot run: 0 
7NI_RX DONE!
Esperando Mensagem!
SendingQueue_front: 2 
API_Try2Send success!
dynamicEnergy_Router >> 3976720 
total instructions~~~> 178871 
TotalEnergy: 11677830 
SendingQueue_front: 3 
API_Try2Send failed - NI_TX occupied!
EnergyPckt sent
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953230 
total instructions~~~> 21066 
TotalEnergy: 4804826 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
Task P4 INIT
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3961238 
total instructions~~~> 99889 
TotalEnergy: 8109693 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3978590 
total instructions~~~> 963742 
TotalEnergy: 43598059 
SendingQueue_front: 7 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
dynamicEnergy_Router >> 3966982 
total instructions~~~> 1000102 
TotalEnergy: 44340787 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3958676 
total instructions~~~> 103626 
TotalEnergy: 8388305 
SendingQueue_front: 2 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3966472 
total instructions~~~> 954551 
TotalEnergy: 42677935 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3962363 
total instructions~~~> 1000203 
TotalEnergy: 44335036 
SendingQueue_front: 5 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
dynamicEnergy_Router >> 3966982 
total instructions~~~> 999838 
TotalEnergy: 44319132 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3958570 
total instructions~~~> 103525 
TotalEnergy: 8387114 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3966472 
total instructions~~~> 963944 
TotalEnergy: 43168698 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3962363 
total instructions~~~> 999900 
TotalEnergy: 44322774 
SendingQueue_front: 3 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
dynamicEnergy_Router >> 3966982 
total instructions~~~> 1000102 
TotalEnergy: 44322762 
SendingQueue_front: 2 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3958570 
total instructions~~~> 103525 
TotalEnergy: 8387313 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 0 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3966472 
total instructions~~~> 963944 
TotalEnergy: 43066516 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
dynamicEnergy_Router >> 3962363 
total instructions~~~> 999900 
TotalEnergy: 44322692 
SendingQueue_front: 1 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
dynamicEnergy_Router >> 3966982 
total instructions~~~> 1000001 
TotalEnergy: 44313960 
SendingQueue_front: 0 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
9NI_RX DONE!
RX interruption catched
10NI_RX DONE!
Mensagem Recebida!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
8NI_RX DONE!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3958676 
total instructions~~~> 103525 
TotalEnergy: 8391477 
SendingQueue_front: 4 
API_Try2Send success!
EnergyPckt sent
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
Esperando Mensagem!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953512 
total instructions~~~> 963944 
TotalEnergy: 41789130 
SendingQueue_front: 6 
API_Try2Send success!
EnergyPckt sent
